MainModule/Server/Shared/Changelog.lua

```diff
@@ -3,18 +3,24 @@ return {
 	"*Drag edges to expand*";
 	"*Report bugs/issues on our GitHub repository*";
 	"";
-	"[v241 2023-12-18 16:00 UTC] @Dimenpsyonal";
-	"Command Box now scrolls (#1307)";
-	":BuyItem now only blocks limiteds (#1223)";
-	"Coasterteam various fixes and improvements (#1242)";
-	"Remove dropper from loader (#1238, #1280)";
-	":clone and :loadavatar are now the same command (#1198)";
-	"Add TextChatService support for command and command autocompletion (#1208)";
-	"Add :sing command (#1217)";
-	"Add new BanMenu UI; add functions+data models to the window code (#1295)";
-	":vote command is now in random order (#1179)";
-	"Add :orderedvote command for voting in order (#1179)";
+	"[v241 2023-12-20 23:59 UTC] Portions (@Dimenpsyonal & @ccuser44)";
```

Comment:
I think this makes it better
```suggestion
	"[v241 2023-12-20 23:59 UTC] Portions (@Dimenpsyonal & @ccuser44 & @moo1210)";
	"The Fixed & Bricked Update";
```